### PR TITLE
Feature/bug fixes

### PR DESCRIPTION
**Pull request by keevkeev**
Broken layout on support page should now be fixed

### CODE REVIEW

File: frontend/scss/components/templates/policies.scss [hunk 1]
```diff
@@ -19,7 +19,7 @@
 @import '../templates/_default.scss';
 
 
-.#{template('code-of-conduct')} {
+.#{template('policies')} {
```
Comment (matthiasrohmer) [inline frontend/scss/components/templates/policies.scss]: `policies` sounds a bit specific. What do you think of `meta`?

File: frontend/templates/views/policies.j2 [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+{{ super() }}
+
+{% endblock %}
+
+{% block main %}
+{% do doc.styles.addCssFile('css/components/templates/policies.css') %}
+<main class="ap--main ap-t-policies">
+  <section class="ap--container">
+    <div class="ap--content-wide">
+      {{ doc.html|render|safe }}
```
Comment (matthiasrohmer) [inline frontend/templates/views/policies.j2]: Don't we have a just-content-template already?

Comment (keevkeev) [reply frontend/templates/views/policies.j2]: none that I could find that isn't loading a lot of unnecessary code.

File: frontend/scss/components/templates/policies.scss (hunk 1 above)
Comment (keevkeev) [reply frontend/scss/components/templates/policies.scss]: sounds good to me 👍

File: frontend/templates/views/partials/footer.j2 [hunk 3]
```diff
@@ -60,7 +60,35 @@
       </ul>
       <div class="ap-o-footer-copyright">&copy; 2019 AMP Open Source Project</div>
     </div>
+
   </div>
+  <amp-position-observer
```
Comment (matthiasrohmer) [inline frontend/templates/views/partials/footer.j2]: To be honest I am not a fan of introducing another instance of `amp-position-observer` and `amp-animation` for this, I think I just dealt with this on another branch by fiddling around with the z-indexes and making the sidebar toggle vanish behind the footer. What do you think?

File: frontend/templates/views/policies.j2 (hunk 2 above)
Comment (matthiasrohmer) [reply frontend/templates/views/policies.j2]: Alright, can we rename to `meta.j2/scss` then? 😉

Comment (keevkeev) [reply frontend/templates/views/policies.j2]: aren't they already renamed or did I miss a file?

Comment (matthiasrohmer) [reply frontend/templates/views/policies.j2]: No all good I was just looking not accurately 🙂 Sorry!

**Pull request merged**